go/engine/bootstrap.go

```diff
@@ -85,12 +88,14 @@ func (e *Bootstrap) Run(ctx *Context) error {
 	if e.G().ConnectivityMonitor.IsConnected(context.Background()) == libkb.ConnectivityMonitorYes {
 		e.G().Log.Debug("connected, running full tracker2 syncer")
 		if err := libkb.RunSyncer(ts, e.status.Uid, false, nil); err != nil {
-			return err
+			e.G().Log.Warning("error running Tracker2Syncer: %s", err)
+			return nil
```

Comment:
I was thinking that it shouldn't, that it would be more beneficial for the app to start if there is an error getting the followers.  But I can go either way.